packages/core/src/events/TapEvent.ts

```diff
@@ -72,7 +70,9 @@ export class TapEvent extends Event {
  * @return true if event was generated by pressing `space` or `enter` on a button
  */
 const isButtonEnterOrSpace = (event: MouseEvent | PointerEvent): boolean => {
-  return isIE && event instanceof PointerEvent ? false : event.detail === 0;
+  // check for click, dbclick, mousedown, mouseup events
+  // as described in https://developer.mozilla.org/en-US/docs/Web/API/UIEvent/detail
+  return event.detail === 0;
```

Comment:
This condition is quite simple, I think we should remove this function and use `event.detail === 0` directly with code comment.

Reply:
Refactored.